rllib/env/single_agent_env_runner.py

```diff
@@ -67,8 +67,6 @@ def __init__(self, config: AlgorithmConfig, **kwargs):
 
         # Create a MetricsLogger object for logging custom stats.
         self.metrics = MetricsLogger()
-        # Initialize lifetime counts.
-        self.metrics.log_value(NUM_ENV_STEPS_SAMPLED_LIFETIME, 0, reduce="sum")
```

Comment:
Don't we need the lifetime counts in the `EnvRunner` anymore? Are they initialized elsewhere?

Reply:
We now call `self.metrics.peek((.. some key))`with arg `default=0`, so no initialization is necessary anymore.